Neos.Media/Configuration/Settings.yaml

```diff
@@ -9,8 +9,7 @@ Neos:
             listener: Neos\Media\Domain\EventListener\ImageEventListener
     mvc:
       routes:
-        Neos.Media:
-          position: 'before Neos.Neos'
+        Neos.Media: true
```

Comment:
How is the behaviour when it is `true`?
Could this lead to an error/validation issue when installing without setting it up properly?

Reply:
The [schema](https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Resources/Private/Schema/Settings/Neos.Flow.mvc.schema.yaml#L35) allows boolean _or_ dictionary and `true` is the default (see https://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartIII/Routing.html#subroutes-from-settings).

With that the ordering depends on the package dependency graph (like it did when specifying an invalid relative position before that led to an exception)